src/modules/10-common/navigation/SideNavV2/ModulesAccordion/ModulesAccordion.module.scss

```
@@ -97,7 +97,7 @@
     border-left: 1px solid var(--grey-200);
   }
 
-  a[data-name='nav-link'] span[data-icon] {
+  a[data-name='nav-link'] > span[class*='bp3-icon'] {
```

```suggestion
  a[data-name='nav-link'] > .bp3-icon {
```
Or pass a custom className if that's possible.

---

@rameshx adding class .bp3-icon directly will not work because it's added by blueprint. 

ModulesAccordian doesn't have access to SideNavLinks, passing custom class is also not possible. 
This code will be removed anyways, we are going to change the design for this.